docs/po/pt.po

```diff
@@ -1590,6 +1590,1003 @@ msgstr "`/blockhash/<ALTURA>`: hash do bloco em uma determinada altura de bloco.
+"generic currency character `¤` should be used."
+msgstr ""
+"O campo `Symbol` é o ponto de código Unicode do símbolo de moeda da rune, que deve ser apresentado após os montantes dessa rune. Se uma rune não tiver um símbolo de moeda, deve "
+"ser usado o carácter genérico de moeda `¤`."
+
+#: src/runes/specification.md:371
+msgid "For example, if the `Symbol` is `#` and the divisibility is 2, the amount of `1234` units should be displayed as `12.34 #`."
+msgstr "Por exemplo, se o `Symbol` é `#` e a divisibilidade é 2, a quantidade de `1234` unidades deve ser apresentada como `12.34 #`."
+
+#: src/runes/specification.md:374
+msgid "Nop"
+msgstr "Nop"
+
+#: src/runes/specification.md:376
+msgid "The `Nop` field is unrecognized."
+msgstr "O campo Nop não é reconhecido."
+
+#: src/runes/specification.md:380
+msgid "Cenotaphs have the following effects:"
+msgstr "Os cenotáfios têm os seguintes efeitos:"
+
+#: src/runes/specification.md:382
+msgid "All runes input to a transaction containing a cenotaph are burned."
+msgstr "Todas as runes introduzidas numa transação que contenha um cenotáfio são queimadas."
+
+#: src/runes/specification.md:384
+msgid "If the runestone that produced the cenotaph contained an etching, the etched rune has supply zero and is unmintable."
+msgstr "Se a runestone que produziu o cenotáfio continha uma gravura, a rune gravada tem fornecimento zero e não pode ser mintada."
+
+#: src/runes/specification.md:387
+msgid "If the runestone that produced the cenotaph is a mint, the mint counts against the mint cap and the minted runes are burned."
+msgstr "Se a runestone que produziu o cenotáfio for um mint, o mint conta para o limite de mints e as runes mintadas são queimadas."
+
+#: src/runes/specification.md:390
+msgid ""
+"Cenotaphs may be created if a runestone contains an unrecognized even tag, an unrecognized flag, an edict with an output number greater than the number of inputs, a rune ID with "
+"block zero and nonzero transaction index, a malformed varint, a non-datapush instruction in the runestone output script pubkey, a tag without a following value, or trailing "
+"integers not part of an edict."
+msgstr ""
+"Cenotáfios podem ser criados se uma runestone contiver uma tag par não reconhecida, uma flag não reconhecida, um édito com um número de saída maior que o número de entradas, "
+"um ID de rune com bloco zero e índice de transação diferente de zero, um varint mal formado, uma instrução non-datapush no script pubkey da saída da runestone, uma tag sem um "
+"valor subsequente, ou inteiros finais que não fazem parte de um édito."
+
+#: src/runes/specification.md:396
+msgid "Executing the Runestone"
+msgstr "Executar a runestone"
+
+#: src/runes/specification.md:398
+msgid "Runestones are executed in the order their transactions are included in blocks."
+msgstr "Os marcos de execução são executados pela `ord`em em que as suas transacções são incluídas nos blocos."
```

Comment:
change all for this:
As runestones são executados na ordem em que as suas transações são incluídas nos blocos.